### DISCUSSION TITLE
能不能禁用循环连接

### DISCUSSION THREAD

**Discussion by Tolerate-yu** [antvis/X6#4419] [Q&A]
![3NT41P139P(JVNS}38TA8GG](https://github.com/user-attachments/assets/46c09e32-d822-47b3-87a4-3663c525c316)
这样的状态

**Comment by zlecheng** [comment]
> ![3NT41P139P(JVNS}38TA8GG](https://private-user-images.githubusercontent.com/103416625/363064546-46c09e32-d822-47b3-87a4-3663c525c316.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUyNDUwMDksIm5iZiI6MTcyNTI0NDcwOSwicGF0aCI6Ii8xMDM0MTY2MjUvMzYzMDY0NTQ2LTQ2YzA5ZTMyLWQ4MjItNDdiMy04N2E0LTM2NjNjNTI1YzMxNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkwMlQwMjM4MjlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zMDViMmVjNDZkNzM0MzBjYTYxNTQ2YTRmZWFmZjJlZDhlNjQwMDVhYjk0OGYyMWVjMGY3OTBkMDRkNjI4ZGMyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.aiCH0xq-u0xUwN4T6n_CwvFlXjTyJsgOrbESzA7I1Dw) 这样的状态

啥意思？

**Reply by Tolerate-yu** [reply]
![image](https://github.com/user-attachments/assets/7c5e391f-69c5-49ef-9f17-4da75d92a8ef)
你看不到图片嘛？图片红框选中的连线方式，就是   当一个节点的不同桩点，连接到下一个节点的不同桩点后；背连接的节点，能循环连接；语言表述比较发杂，你看图片

**Reply by Tolerate-yu** [reply]
我想禁用这样的连接方式

**Comment by resetsix** [comment]
https://x6.antv.antgroup.com/api/model/interaction#allowloop
